addons/hw_drivers/tools/helpers.py

```diff
@@ -80,6 +80,22 @@ def writable():
                 subprocess.run(["sudo", "mount", "-o", "remount,ro", "/root_bypass_ramdisks/"], check=False)
                 subprocess.run(["sudo", "mount", "-o", "remount,rw", "/root_bypass_ramdisks/etc/cups"], check=False)
 
+
+def require_db(function):
+    """Decorator to check if the IoT Box is connected to a server before
+    executing the function
+    """
+    fname = f"<function {function.__module__}.{function.__qualname__}>"
```

Comment:
if the only usage is in the logger, init the variable there (or just call it directly as it is used once)